src/test/functional/features/sole-hub-page.feature

```diff
@@ -132,6 +132,11 @@ Feature: Sole hub page
     When I login with applicant "1"
     Then the page should include "Your application has been received and will be reviewed by a judge. You will receive an email telling you whether your application has been successful."
 
+    Given I set the case state to "BailiffRefused"
+    And I click "Sign out"
+    When I login with applicant "1"
+    Then the page should include "Your application has been received and will be reviewed by a judge. You will receive an email telling you whether your application has been successful."
```

Comment:
Are we not updating the hub page based on the outcome of Bailiff application?

Reply:
I checked this with Kim in the ticket comments and she seems happy to leave the hub page as it is when it moves to the new state.

The idea was that BailiffRefused is a new intermediate state after a legal advisor refuses bailiff service. Caseworkers can filter cases by this state and do some admin tasks before confirming the bailiff refusal. When the refusal is confirmed, we send bailiff refusal notifications, the case transitions to AwaitingAos and the hub page changes.